apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2Test.java

```diff
@@ -641,4 +663,92 @@ public void addCareTeamMemberWithNpiOrgShouldCreateCareTeamEntry() {
         "National Provider Identifier",
         careTeamEntry.getProvider().getIdentifier().getType().getCoding().get(0).getDisplay());
   }
+
+  /**
+   * Verifies that {@link gov.cms.bfd.server.war.r4.providers.TransformerUtilsV2#createBundle
+   * (OffsetLinkBuilder paging, List<IBaseResource> resources, Instant transactionTime)} sets bundle
+   * size correctly} correctly.
+   */
+  @Test
+  public void createBundleWithoutPaging() throws IOException {
+
+    RequestDetails requestDetails = mock(RequestDetails.class);
+    OffsetLinkBuilder paging = new OffsetLinkBuilder(requestDetails, "/ExplanationOfBenefit?");
+
+    List<Object> parsedRecords =
+        ServerTestUtils.parseData(Arrays.asList(StaticRifResourceGroup.SAMPLE_A.getResources()));
+
+    HHAClaim claim =
+        parsedRecords.stream()
+            .filter(r -> r instanceof HHAClaim)
+            .map(r -> (HHAClaim) r)
+            .findFirst()
+            .get();
+
+    claim.setLastUpdated(Instant.now());
+
+    FhirContext fhirContext = FhirContext.forR4();
+    ExplanationOfBenefit genEob =
+        HHAClaimTransformerV2.transform(
+            new TransformerContext(
+                new MetricRegistry(),
+                Optional.empty(),
+                FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting(),
+                NPIOrgLookup.createNpiOrgLookupForTesting()),
+            claim);
+    IParser parser = fhirContext.newJsonParser();
+    String json = parser.encodeResourceToString(genEob);
+    List<IBaseResource> eobs = new ArrayList<IBaseResource>();
+    eobs.add(parser.parseResource(ExplanationOfBenefit.class, json));
+
+    Bundle bundle = TransformerUtilsV2.createBundle(paging, eobs, Instant.now());
+    assertEquals(1, bundle.getTotal());
+  }
+
+  /**
+   * Verifies that {@link gov.cms.bfd.server.war.r4.providers.TransformerUtilsV2#createBundle
+   * (OffsetLinkBuilder paging, List<IBaseResource> resources, Instant transactionTime)} sets bundle
+   * with paging size correctly} correctly.
+   */
+  @Test
+  public void createBundleWithPaging() throws IOException {
+
+    RequestDetails requestDetails = mock(RequestDetails.class);
+    Map<String, String[]> pagingParams = new HashMap<String, String[]>();
+    pagingParams.put(Constants.PARAM_COUNT, new String[] {"1"});
+    pagingParams.put("startIndex", new String[] {"1"});
```

Comment:
Is this only making one page? Shouldnt there be multiple pages to properly test this works with paging?